src/no-start-duplicated-conjunction.js

```diff
@@ -55,7 +57,11 @@ module.exports = function(context, options = {}) {
                 }
                 if (useDuplicatedPhase) {
                     const sentenceStartIndex = node.range[0] + sentence.range[0];
-                    if (!ignoreNodeManager.isIgnoredIndex(sentenceStartIndex)) {
+                    const originalIndex = source.originalIndexFromIndex(sentenceStartIndex);
+                    // adjust index 
+                    // if  "また、[import, a.js](a.js)" then originalIndex is used.
+                    // if "[import, binary-example.js](src/binary-example.js)" then  originalIndex is undefined.
+                    if (!ignoreNodeManager.isIgnoredIndex(typeof originalIndex !== "undefined" ? originalIndex : sentenceStartIndex)) {
```

Comment:
> typeof originalIndex !== "undefined" ? originalIndex : sentenceStartIndex

It is better that assign to variable like following:

```js
const targetIndex = typeof originalIndex !== "undefined" ? originalIndex : sentenceStartIndex
```

Reply:
Thanks for the review.
I think I will fix it.